### DISCUSSION TITLE
Isoforms vs. Genes

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1383] [Q&A] [answered]
Hi all,

Alright, this may be a pretty significant question. Back when I started on my analysis of the Tanner crab (and the _Hematodinium_) transcriptome and expression patterns, I wasn't too knowledgeable about the notation of Trinity, and just assumed each line in my kallisto files was one gene.

However, as I'm going back through things, I realize that each line was actually one **isoform**, not a gene. This really impacts all my downstream analyses, as expression of each individual isoform is treated as its own gene. 

Basically, my question is: **when performing differential expression analyses, is it alright to use isoforms rather than genes?**

**Comment by sr320** [comment, marked as answer]
Yes 

ps transcript would be better term than isoform.

**Reply by aspencoyle** [reply]
Gotcha. So just to confirm and remove any ambiguity here, I did *not* mess up monumentally badly and do *not* need to redo a huge portion of my analysis, right?

**Reply by sr320** [reply]
correct 
unless there is a complete genome in place - analysis is often done at assembled transcript level
